Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6994.cs

```diff
@@ -74,7 +74,9 @@ protected override void Init()
 						await Task.Delay(20);
 						source.Clear();
 						await Task.Delay(2000);
+#pragma warning disable CS4014
 						Device.InvokeOnMainThreadAsync(() => instructions.Text = "Success");
```

Comment:
I think this was intentional, so I disabled the warning, if not, await it :) @samhouts